packages/contracts/src/utils/RelayRegistrar.sol

```diff
@@ -97,6 +111,21 @@ contract RelayRegistrar is IRelayRegistrar, ERC165 {
 
     /// @inheritdoc IRelayRegistrar
     function readRelayInfos(
+        address relayHub
+    )
+    public
+    view
+    override
+    returns (
+        RelayInfo[] memory info
+    ) {
+        uint256 blockTimestamp = block.timestamp;
+        uint256 oldestBlockTimestamp = blockTimestamp >= relayRegistrationMaxAge ? blockTimestamp - relayRegistrationMaxAge : 0;
+        return readRelayInfosInRange(relayHub, 0, oldestBlockTimestamp, 100);
```

Comment:
please make a constant, and set to a real large value (that is still good gas-wise, e.g. 1000)